todo-app/src/TodoList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+import todoItems from './todo.json';
+
+export default class TodoList extends React.Component {
+
+  state = {
+    todos: todoItems
+  }
+    
+  onDelete = (todoID) => {
+    var updatedTodos = this.state.todos.filter((val,index) =>{
+    return val.id !== todoID;
+    })
+    this.setState({
+      todos: updatedTodos
+    })  
+  }
+  
+  onDone = (todoID) => {
+    this.setState({
+      todos: this.state.todos.map(todo => {
+        if (todo.id === todoID) {
+          todo.done = !todo.done
+        }
+        return todo
+      })
+    })
+  } 
+ 
+  render() {
+    const { todos } = this.state
+    if(todos.length == 0){
```

Comment:
Careful here. If you run your app and open the Chrome devtools, you'll see this warning:

    webpackHotDevClient.js:241 ./src/TodoList.js
    Line 33:  Expected '===' and instead saw '=='  eqeqeq

You should always be using "strict equality comparison" unless there's a good reason not to. Here's a [StackOverflow question](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons) about it, and there's a bunch of info on the topic.